ports/crashpad/vcpkg.json

```
@@ -1,19 +1,30 @@
 {
   "name": "crashpad",
-  "version-date": "2022-04-16",
-  "port-version": 1,
+  "version-date": "2022-04-27",
   "description": [
     "Crashpad is a crash-reporting system.",
     "Crashpad is a library for capturing, storing and transmitting postmortem crash reports from a client to an upstream collection server. Crashpad aims to make it possible for clients to capture process state at the time of crash with the best possible fidelity and coverage, with the minimum of fuss."
   ],
   "homepage": "https://chromium.googlesource.com/crashpad/crashpad/+/master/README.md",
   "license": "Apache-2.0",
-  "supports": "osx | windows",
+  "supports": "static & (((x64 | x86 | arm64) & windows) | (x64 & (osx | linux)) | android)",
```

Generally ports should not be marked as only supporting static libraries -- vcpkg policy allows ports to ignore dynamic lib requests by instead calling the port helper `vcpkg_check_linkage(ONLY_STATIC_LIBRARY)`.

We strongly prefer known-bad lists over known-good lists since they enable users to work through configurations we may not know about. If possible, it would be great to change this conditional to block known-bad configurations instead of trying to enumerate every known-good configuration. As an example: do we know that iOS doesn't work? Do we know that MacOS arm64 doesn't work?

```suggestion
  "supports": "((x64 | x86 | arm64) & windows) | (x64 & (osx | linux)) | android",
```